common/primitives/src/capacity.rs

```diff
@@ -24,7 +24,7 @@ pub trait Nontransferable {
 	fn balance(msa_id: MessageSourceId) -> Self::Balance;
 
 	/// Reduce Capacity of an MSA account by amount.
-	fn withdraw(msa_id: MessageSourceId, amount: Self::Balance) -> Result<(), DispatchError>;
+	fn deduct(msa_id: MessageSourceId, amount: Self::Balance) -> Result<(), DispatchError>;
```

Comment:
FYI, in the fungible trait that parity is swapping to, they call it decrease_balance. I am good with this thought.

https://github.com/paritytech/substrate/blob/master/frame/support/src/traits/tokens/fungible.rs#L349